lib/styles/v2/components/_tag.scss

```diff
@@ -0,0 +1,43 @@
+$solid-indigo: #5E5CE6;
+$solid-indigo-bg: #EAF3FC;
+
+.nui-tag {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  color: theme("colors.gray.800");
+  line-height: 22px;
+  font-size: 12px;
+  padding: 0 12px;
+  height: 22px;
+  background: white;
+  border: 1px solid theme("colors.gray.400");
+  border-radius: 3px;
```

Comment:
1. Use variables for font size, color and border radius.
2. Follow the same padding, line-height etc as in the design.
3. We shouldn't use fixed height unless necessary.